src/test/java/uk/gov/pay/ledger/report/dao/PerformanceReportDaoIT.java

```diff
@@ -64,7 +72,8 @@ public void report_volume_total_amount_and_average_amount_for_date_range() {
 
     @Test
     public void report_volume_total_amount_and_average_amount_by_state() {
-        aTransactionFixture().withAmount(1000L).withState(TransactionState.STARTED).withTransactionType("PAYMENT")
+        aTransactionFixture()
+                .withAmount(1000L).withState(TransactionState.STARTED).withTransactionType("PAYMENT")
```

Comment:
it doesn't look like this change is applicable to this PR...?

Reply:
I wrote it to verify that the SQL didn't return transactions with transaction status of STARTED - happy to remove otherwise?

Reply:
hmm no it seems like only a line break was added. the test `report_volume_total_amount_and_average_amount_by_state` isn't applicable to the PR.

Reply:
Oh apologies I must of scrolled up and accidentally modified that when I added the line breaks to my test!

Reply:
can you revert this change then please?

Reply:
Done